aliyun/src/test/java/org/apache/iceberg/aliyun/oss/mock/AliyunOSSMockLocalStore.java

```diff
@@ -87,7 +86,7 @@ static String md5sum(InputStream is) throws IOException {
     while ((numBytes = is.read(bytes)) != -1) {
       md.update(bytes, 0, numBytes);
     }
-    return new String(Hex.encodeHex(md.digest())).toUpperCase(Locale.ROOT);
+    return Hex.encodeHexString(md.digest(), false);
```

Comment:
why is this changed?

Reply:
`org.apache.directory.api.util.Hex` is not available after switching to Hadoop 3. Also, I think it is more reasonable to use a function from Apache Commons for this purpose.